pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -285,3 +286,10 @@ func getPreScoreState(cycleState *framework.CycleState) (*preScoreState, error)
 func topologyNormalizingWeight(size int) float64 {
 	return math.Log(float64(size + 2))
 }
+
+func adjustForMaxSkew(cnt, maxSkew int64) int64 {
+	if cnt < maxSkew {
```

Comment:
needs a comment to explain the rational, something like "topologies with less than maxSkew pods are considered to have the same priority".

Reply:
Done